### PR TITLE
tests: avoid deprecated `kubectl run` in secrets scenario

### PR DESCRIPTION
**Pull request by spadgett**
and fix bug in monitoring scenario.

### CODE REVIEW

**Comment by spadgett** [comment]
/assign @rhamilto @benjaminapetersen 

This should unblock CI

**Comment by spadgett** [comment]
I added some clean up logic since we weren't deleting the resources. @rhamilto another look?

**Comment by andrewballantyne** [comment]
`ci/prow/analyze` failed because of output size? 

> FAILURE: Main vendor bundle is larger than the 2 MiB limit.

**Comment by benjaminapetersen** [comment]
hmm. that analyze job has been quirky as well.

**Comment by andrewballantyne** [comment]
Seems like a webpack issue? How'd this ever get in if there is a dependency bloating issue?

> Main vendor bundle size: 3.85 MiB
FAILURE: Main vendor bundle is larger than the 2 MiB limit.
If you haven't added a new dependency, an import might have accidentally pulled an existing dependency into the main vendor bundle.
If adding a large dependency, consider lazy loading the component with AsyncComponent.

**Comment by spadgett** [comment]
@andrewballantyne I've seen `yarn run analyze` give different results different times you run it with no other code changes :(

Seems to be from the PF icons:

https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_console/3847/pull-ci-openshift-console-master-analyze/2785/artifacts/analyze/report.html

**Comment by andrewballantyne** [comment]
@spadgett perhaps I am misunderstanding the build process here... but if it sometimes packages a larger set of icons, wouldn't this run the risk of an actual release being bloated? Or do we run a similar / the same test against the release output?

I am also confused how packaging could produce a non-deterministic result.

**Comment by spadgett** [comment]
Yeah, it's alarming. I'm worried about this as well. We need to get to the bottom of it.

File: frontend/integration-tests/tests/storage.scenario.ts [hunk 1]
```diff
@@ -54,6 +54,7 @@ describe('Add storage is applicable for all workloads', () => {
         expect($(`[data-id="${pvcName}-${mountPath}"] [data-test-id="path"]`).getText()).toContain(
           mountPath,
         );
+        execSync(`kubectl delete --cascade ${resourceType} example -n ${testName}`);
```
Comment (andrewballantyne) [inline frontend/integration-tests/tests/storage.scenario.ts]: @spadgett This appears to be a double delete, and the other case has a try..catch; thoughts?

https://github.com/openshift/console/pull/3847/files#diff-cbca0dc593db54f1952a611b25cf49d0R27
```
    resourceObjs.forEach((resourceType) => {
      try {
        execSync(`kubectl delete --cascade ${resourceType} example -n ${testName}`);
      } catch (error) {
        console.error(`Failed to delete ${resourceType} example:\n${error}`);
      }
    });
```

Comment (spadgett) [reply frontend/integration-tests/tests/storage.scenario.ts]: I added that temporarily to debug something. Let me add WIP to the title

**Comment by spadgett** [comment]
Got all the way to the Alertmanager tests, which flaked.

/retest

**Pull request merged**